xpdtools/pipelines/masking.py

```diff
@@ -0,0 +1,60 @@
+import numpy as np
+from streamz_ext import Stream
+
+from xpdtools.tools import mask_img
+
+mask_setting = {"setting": "auto"}
+
+
+def make_pipeline():
```

Comment:
will these be used as ``masking.make_pipeline()`` and ``image_preprocess.make_pipeline()``, or do we need more unique naming here?  I think I like the former, so that things are uniform between modules.  In that case should ``binner_pipeline()`` be ``integration.make_pipeline()``?

not sure if any of this is relevant because I haven't tried to use these tools yet.  Perhaps we should have a coding sprint to do this?  then we can give better feedback on usability.

Reply:
We may need more unique names, since we might consider moving the median and std integrations into the integration module, since we no longer build the pipeline at import time it will be safe.